### DISCUSSION TITLE
关于speaker增加的问题

### DISCUSSION THREAD

**Discussion by shanhaidexiamo** [lenML/Speech-AI-Forge#17] [问答 | Q&A]
hi，我想咨询下data 路径下speaker增加的方法。是使用了VQ Encoder 将speaker的语音转换成了embedding并保存为pt文件吗？还是有其他别的方法？谢谢~

**Comment by zhzLuke96** [comment]
你可以从 `/v1/speaker/create` 这个接口用 seed 值创建 speaker，创建后会保存到 data/speakers 下面
pt文件里除了 embedding 还有一些元信息

不清楚你指的其他方法是指的什么？语音克隆？目前 chattts 还没有方便的克隆方法，我们这里也不准备实现

这部分还在完善，后面可能还会增加从 tensor 数组创建 speaker
